src/TLuaInterpreter.cpp

```diff
@@ -6287,6 +6287,7 @@ std::pair<int, QString> TLuaInterpreter::startPermRegexTrigger(const QString& na
     pT->setIsFolder(patterns.empty());
     pT->setIsActive(true);
     pT->setTemporary(false);
+    pT->setIsMultiline(multiline);
```

Comment:
The line on 6278 should be revised - and indeed the `TTrigger` constructor used there should be modified completely to remove the fourth argument - which is used to set the multi-line mode but which would only be required in the future for the one (of the eleven) uses of that constructor (in `TLuaInterpreter::tempComplexRegexTrigger(...)`) and which could reasonably be done with a line like this there.

This applies to all the Lua API trigger functions revised herein.

Reply:
I don't think I follow what you are saying.

Reply:
The fourth argument to the `TTrigger(const QString&, const QStringList&, const QList<int>&, bool, Host*)` constructor IS used to set the `(bool) mIsMultiline` in the constructor list - so there is no need to set it in that separate `pT->setIsMultiLine(bool)` call if you do it in the contructor. OTOH if you retain that separate call then there is no need to have it set by that constructor.